### PR TITLE
[Bug fix ] - An issue in the 'MDE - False Positive Incident Handling'  

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3892)

## Description
Changed the `indicatorsValues` input in the 'Tag Indicators' task:

 **From** `${Indicators to Block.Questions.0}` **to** `${Indicators to Allow.Answers.0}`.

## Screenshots
N.a.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_12_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### MDE - False Positive Incident Handling
+Fixed typo misspelling in the 'Tag Indicators' task (Changed the 'indicatorsValues' input to "...Allow.Answers.0").
```
Comment (altmannyarden) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_12_1.md:4]: what's the typo? Seems like you are fixing an issue with tag indicator input.

Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_12_1.md:4]: Indeed this is an issue, which I addressed as a typo. Would you like to change that to just as an issue?

Comment (altmannyarden) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_12_1.md:4]: yes please :)

Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_12_1.md]: Done!

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**